main.py

```diff
@@ -269,6 +269,7 @@ def enable_switch(self, dpid):
 
         self.notify_switch_enabled(dpid)
         self.notify_topology_update()
+        self.notify_switch_links_status(switch, "link enable")
```

Comment:
@Alopalao, minor stuff, but the reason name for `enable` and `disable` this NApp has been used the past tense verbse, so for consistency in the event content, could you replace with [`link enabled`](https://github.com/kytos-ng/topology/blob/master/main.py#L479) and [`link disabled`](https://github.com/kytos-ng/topology/blob/master/main.py#L496) accordingly?

Reply:
Sure.